### ISSUE TITLE
XSL not working in Dreamweaver

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#260]
```
What steps will reproduce the problem?
1. Open .xsl document
2. Type tm and tab

What is the expected output? What do you see instead?
Results in <tm></tm>. Should result in <xsl:template match="" 
mode=""></xsl:template>

What version of the product are you using? On what operating system?

Dreamweaver CS5 with Zen Coding v0.7.2 on Windows 7 64bit

Please provide any additional information below.

Other languages expand, just not XSL.
```

---

Original issue reported on code.google.com by `m...@casadelewis.com` on 3 Apr 2011 at 5:39

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `serge....@gmail.com` on 13 Apr 2011 at 7:03

**Comment by GoogleCodeExporter** [comment]
```
My bad. Dreamweaver recognizes XSLT, but result was wrote to wrong variable. 
Fixed in version 0.7.5.
```

---

Original comment by `GreLIm...@gmail.com` on 21 May 2011 at 6:58
- Changed state: **Fixed**
